packages/formvuelate/src/SchemaFormFactory.js

```diff
@@ -22,6 +22,18 @@ export default function SchemaFormFactory (plugins = [], components = null) {
     }
   }
 
+  const formProps = { ...SchemaForm.props }
```

Comment:
suggestion: I dont know why but `formProps` just seems confusing, how about `schemaFormProps` to signify that its the original SchemaForm props here